src/commands/deployments/disconnectRepo.ts

```diff
@@ -3,13 +3,21 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DeploymentsTreeItem } from "vscode-azureappservice";
+import { DeploymentsTreeItem, disconnectRepo as disconnectRepository } from "vscode-azureappservice";
 import { IActionContext } from "vscode-azureextensionui";
 import { ext } from "../../extensionVariables";
+import { localize } from '../../localize';
+import { SlotTreeItemBase } from '../../tree/SlotTreeItemBase';
 
 export async function disconnectRepo(context: IActionContext, node?: DeploymentsTreeItem): Promise<void> {
     if (!node) {
         node = await ext.tree.showTreeItemPicker<DeploymentsTreeItem>(DeploymentsTreeItem.contextValueConnected, context);
     }
-    await node.disconnectRepo(context);
+
+    if (node.parent instanceof SlotTreeItemBase) {
+        await disconnectRepository(context, node.parent.client, node.parent.root);
+        await node.refresh();
```

Comment:
I didn't see any, but are there any checks I need to do before refreshing?

Reply:
Don't think so